Libplanet.Explorer/Queries/VoteQuery.cs

```diff
@@ -0,0 +1,246 @@
+#nullable disable
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL;
+using GraphQL.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Consensus;
+using Libplanet.Crypto;
+using Libplanet.Explorer.GraphTypes;
+using Libplanet.Explorer.Interfaces;
+using Libplanet.Store;
+
+namespace Libplanet.Explorer.Queries
+{
+    public class VoteQuery<T> : ObjectGraphType
+        where T : IAction, new()
+    {
+        public VoteQuery(IBlockChainContext<T> chainContext)
```

Comment:
```suggestion
        public VoteQuery(BlockChain<T> blockChain, IStore store)
```

In most cases, `IBlockChainContext<T>` isn't necessary.  😅